verifier/src/advisories.rs

```diff
@@ -40,11 +40,12 @@ pub enum AdvisoryStatus {
     /// enclaves may be needed.
     SWHardeningNeeded,
     /// TCB level of the SGX platform is up-to-date.
+    #[default]
     UpToDate,
 }
 
 /// The advisories pertaining to a TCB(Trusted Computing Base).
-#[derive(Debug, PartialEq, Eq, Hash, Clone)]
+#[derive(Debug, Default, Clone, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
```

Comment:
Do you know where the `Ord` and `PartialOrd` are needed at?
This allows someone to do `advisories_1 < advisories_2` which for verification purposes would be good to avoid.

Reply:
They are needed for MrEnclaveVerifier and MrSignerVerifier along with the Kind enum in the mc repo. But it appears that those structs don't need the traits, so I'll remove it all the way down